models/machikoro_game.cpp

```diff
@@ -61,4 +66,80 @@ std::unique_ptr<Event> MachiKoroGame::initGame()
     log_->info(event->message() + "Game ID: " + game_id_);
 
     return event;
+}
+
+std::unique_ptr<Event> MachiKoroGame::rollDice(const std::string& player_name, int dice_count)
+{
+    //Get the order of the player
+    int player_order = getPlayerIndex(player_name);
+
+    if (player_order == -1 || player_order != current_player_)
```

Comment:
When to change the `current_player_` ?

Reply:
For now, we only implement the roll_dice feature. We will change the current_player_ to the next one once we finish the implemetation of the whole round (rolldice -> buy cards -> ...)